app/models/user.rb

```diff
@@ -0,0 +1,33 @@
+require 'bcrypt'
+require 'dm-validations'
+
+class User
+  include DataMapper::Resource
+  include BCrypt
+
+  property :id, Serial
+  property :name, String, :required => true
+  property :username, String, :required => true, :unique => true
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)